lib/plugins/create/templates/aws-nodejs-typescript/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Serverless aws-nodejs-typescript template",
   "main": "serverless.ts",
   "scripts": {
+    "build": "serverless webpack",
```

Comment:
If you're adding this command to build lambda code, could you document it in the readme ?

Reply:
I've removed it to avoid confusion for other project build steps, eg `tsc`